ocean_lib/common/ddo/credentials.py

```diff
@@ -0,0 +1,106 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+from ocean_lib.common.agreements.consumable import ConsumableCodes, MalformedCredential
+
+
+class AddressCredential:
+    def __init__(self, asset):
+        self.asset = asset
+
+    def get_addresses_of_class(self, access_class="allow"):
```

Comment:
Refactor based on the comment for `get_list_entry_with_type_address`

Reply:
Resolving because this comment has become stale. I'll mention it in my next review.